tests/samples/bluetooth/samples_test_app/README.rst

```diff
@@ -0,0 +1,231 @@
+.. _samples_test_app:
+
+Bluetooth: Bluetooth LE master test app
+#######################################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The Bluetooth LE master test app is a Bluetooth® Low Energy (LE) central application designed to test basic Bluetooth functionality in peripheral samples.
+This application acts as a tester in the central role to validate Bluetooth LE peripheral applications in the |NCS|.
+
+Requirements
+************
+
+The sample supports the following development kits:
```

Comment:
Generally, we are mixing the term test app with sample in the doc in multiple instances. Could be good to fix that and consistently use 'test app'. In the building and running section, we might need new shortcuts defined for test (just like for sample)  also if we have to replace 'sample' with 'test'  or 'test app'.